pontoon/base/views.py

```diff
@@ -1224,20 +1229,21 @@ def save_to_transifex(request, template=None):
 
 
 @login_required(redirect_field_name='', login_url='/403')
+@require_POST
 @transaction.atomic
-def quality_checks_switch(request):
-    """Turn quality checks on/off for the current user."""
-    log.debug("Turn quality checks on/off for the current user.")
+def toggle_user_profile_attribute(request, email):
+    user = get_object_or_404(User, email=email)
```

Comment:
I'm pretty sure this means any user can change the `quality_checks` and `force_suggestions` preferences of any other user. You'll want to ensure that the current user has permission to edit this user (for now just checking that they're equal is fine, no need for full-blown permissions yet).